src/components/common/NotFixedHeaderDetail.tsx

```
@@ -0,0 +1,39 @@
+
+export const NotFixedHeaderDetail: React.FC<TopBarProps> = ({
+  title,
+  onBackClick,
+  onNotificationClick
+}) => {
+  return (
+    <header className="top-0 px-[18px] flex z-40 justify-between items-center bg-white border-b border-solid border-b-black-300 h-[60px] w-full max-w-[clamp(360px,100vw,430px)]">
+      <button onClick={onBackClick} aria-label="뒤로 가기">
+        <div
+          dangerouslySetInnerHTML={{
+            __html:
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)